Description: Fix libdir in qws qmake .pro files.
  By default the qws c-to-haskell wrapper of the qtHaskell installs
  it's libraries into /usr/local/lib directory which is not very
  nice if it would remain in the package like that.
Author: Filip Brcic <brcha@gna.org>
Origin: vendor
Last-Update: 2011-05-23

Index: qthaskell-1.1.4.1/qws/qtc_gui/qtc_gui.pro
===================================================================
--- qthaskell-1.1.4.1.orig/qws/qtc_gui/qtc_gui.pro	2011-05-24 01:27:29.000000000 +0200
+++ qthaskell-1.1.4.1/qws/qtc_gui/qtc_gui.pro	2011-05-24 02:02:50.125116613 +0200
@@ -30,6 +30,6 @@
   target = qtc_gui
 }

-target.path = /usr/local/lib
+target.path = /usr/lib
 INSTALLS += target

Index: qthaskell-1.1.4.1/qws/qtc_script/qtc_script.pro
===================================================================
--- qthaskell-1.1.4.1.orig/qws/qtc_script/qtc_script.pro	2011-05-24 01:27:29.000000000 +0200
+++ qthaskell-1.1.4.1/qws/qtc_script/qtc_script.pro	2011-05-24 02:02:50.135116613 +0200
@@ -30,6 +30,6 @@
   target = qtc_script
 }

-target.path = /usr/local/lib
+target.path = /usr/lib
 INSTALLS += target

Index: qthaskell-1.1.4.1/qws/qtc_tools/qtc_tools.pro
===================================================================
--- qthaskell-1.1.4.1.orig/qws/qtc_tools/qtc_tools.pro	2011-05-24 01:27:29.000000000 +0200
+++ qthaskell-1.1.4.1/qws/qtc_tools/qtc_tools.pro	2011-05-24 02:02:50.135116613 +0200
@@ -30,6 +30,6 @@
   target = qtc_tools
 }

-target.path = /usr/local/lib
+target.path = /usr/lib
 INSTALLS += target

Index: qthaskell-1.1.4.1/qws/qtc_core/qtc_core.pro
===================================================================
--- qthaskell-1.1.4.1.orig/qws/qtc_core/qtc_core.pro	2011-05-24 01:27:29.000000000 +0200
+++ qthaskell-1.1.4.1/qws/qtc_core/qtc_core.pro	2011-05-24 02:02:50.125116613 +0200
@@ -36,6 +36,6 @@
   target = qtc_core
 }

-target.path = /usr/local/lib
+target.path = /usr/lib
 INSTALLS += target

Index: qthaskell-1.1.4.1/qws/qtc_network/qtc_network.pro
===================================================================
--- qthaskell-1.1.4.1.orig/qws/qtc_network/qtc_network.pro	2011-05-24 01:27:29.000000000 +0200
+++ qthaskell-1.1.4.1/qws/qtc_network/qtc_network.pro	2011-05-24 02:02:50.135116613 +0200
@@ -30,6 +30,6 @@
   target = qtc_network
 }

-target.path = /usr/local/lib
+target.path = /usr/lib
 INSTALLS += target

Index: qthaskell-1.1.4.1/qws/qtc_opengl/qtc_opengl.pro
===================================================================
--- qthaskell-1.1.4.1.orig/qws/qtc_opengl/qtc_opengl.pro	2011-05-24 01:27:29.000000000 +0200
+++ qthaskell-1.1.4.1/qws/qtc_opengl/qtc_opengl.pro	2011-05-24 02:02:50.135116613 +0200
@@ -32,6 +32,6 @@
   target = qtc_opengl
 }

-target.path = /usr/local/lib
+target.path = /usr/lib
 INSTALLS += target
